database/datagenerator.js

```diff
@@ -0,0 +1,59 @@
+const faker = require('faker');
+const data = require('fs').createWriteStream('database/files/data.csv');
+const images = require('fs').createWriteStream('database/files/images.csv');
+const tags = require('fs').createWriteStream('database/files/tags.csv');
+const types = require('fs').createWriteStream('database/files/types.csv');
+
+const generateData = async (callback) => {
+    
+    const cuisine = ['Japanese', 'Korean', 'Chinese', 'French', 'German', 'American', 'Mexican',
+                    'Vietnamese', 'Taiwanese', 'Brazilian', 'Italian', 'Martian', 'Connecticotian',
+                     'Indian', 'Jamaican', 'Seafood', 'Ethiopian', 'Mediterranean', 'Californian'];
+    // write headers for every csv file
+    data.write('id,name,description,reviews,priceStart,priceEnd\n');
+    images.write('id,listingId,imgUrl,caption\n');
+    tags.write('id,listingId,tag\n');
+    types.write('id,listingId,type\n');
+    // create 10m records 
+    for(let id = 1; id <= 1e7; id++) {
+        // generate random number for photo, tags and cuisines
+        const randPhotoNum = Math.floor(Math.random() * 35 + 10);
+        const randTagNum = Math.floor(Math.random() * 10 + 3);
+        const randCuiNum = Math.floor(Math.random() * 2 + 1);
+        // create a random data with faker
+        const post = `${id},${faker.random.word()},${faker.lorem.paragraphs(3)},
+                      ${ faker.random.number({min: 1, max: 999})},${faker.random.number({min: 2, max: 5, precision: 0.01})},
```

Comment:
there's a space before the first faker, keep spacing consistent

Reply:
personally, i think spaces after the opening bracket and before the closing bracket works best